packages/application-components/src/components/modal-pages/internals/modal-page.styles.js

```diff
@@ -0,0 +1,53 @@
+import { css, keyframes } from '@emotion/core';
+import styled from '@emotion/styled';
+import { customProperties } from '@commercetools-frontend/ui-kit';
+
+const slideFromLeft = keyframes`
```

Comment:
Didn't we say to keep the animation disabled for now?

Reply:
I've talked with Sven and also showed the animation to Filip here in Berlin and they were quite happy about it and decided it should stay 🙂